tools/pkglint/lib/rules.ts

```diff
@@ -1372,6 +1372,11 @@ export class YarnNohoistBundledDependencies extends ValidationRule {
   public readonly name = 'yarn/nohoist-bundled-dependencies';
 
   public validate(pkg: PackageJson) {
+    // don't do this check for the separately-vended modules
+    if (pkg.packageName.startsWith('@aws-cdk-lib-alpha/')) {
+      return;
+    }
```

Comment:
Why is this being skipped?

Reply:
Huh, it's not clear. I removed this check and re-ran pkglint, and didn't see any differences in any of these packages. Perhaps a hold-out from an older iteration? I'll remove for now until the reason becomes clearer.

Reply:
Huh, I must not have built and/or linted the right thing. Here's the build failure.

```
@aws-cdk-lib-alpha/assertions: In package package.json
@aws-cdk-lib-alpha/assertions: - [yarn/nohoist-bundled-dependencies] Repository-level 'workspaces.nohoist' directive is missing: @aws-cdk-lib-alpha/assertions/colors, @aws-cdk-lib-alpha/assertions/colors/**, @aws-cdk-lib-alpha/assertions/diff, @aws-cdk-lib-alpha/assertions/diff/**, @aws-cdk-lib-alpha/assertions/fast-deep-equal, @aws-cdk-lib-alpha/assertions/fast-deep-equal/**, @aws-cdk-lib-alpha/assertions/string-width, @aws-cdk-lib-alpha/assertions/string-width/**, @aws-cdk-lib-alpha/assertions/table, @aws-cdk-lib-alpha/assertions/table/** (fixable)
```

Reply:
I don't know enough to understand this. What is this rule meant to prevent and why we should not apply this to v2?

For example, an alternate solution could be to add this to the 'nohoist' flag here - https://github.com/aws/aws-cdk/blob/v2-main/packages/individual-packages/lerna.json. 
I'm not saying that's the right solution, just that I don't understand enough of it.

Can you provide a little more context?

Reply:
It's really not clear. I don't think I understand why Adam wanted to exclude the alpha packages from this rule.

The build is complaining that a bundled dependency (e.g., colors) is not added to the top-level `nohoist` config, which keeps those dependencies at the local package level rather than hoisting them up to the top-level workspace. We have colors as a nohoist listing three other times already (`@aws-cdk/assertions/colors`, `aws-cdk-lib/colors`, and `monocdk/colors`). Not sure why we wouldn't add the alpha packages in the same way.

I ran pkglint -f in the alpha assertions package, which added the relevant entries to the top-level package.json. This should solve the build error, and I think is the right thing to do.

Reply:
Thanks for diving in. Sounds like the right approach, until we have info that says otherwise.